addons/purchase_requisition/views/purchase_views.xml

```diff
@@ -38,7 +38,10 @@
                             <field name="partner_id" readonly="state in ['cancel', 'done', 'purchase']"/>
                             <field name="name" string="Reference"/>
                             <field name="date_planned"/>
-                            <field name="amount_total"/>
+                            <field name="currency_id" column_invisible="True"/>
```

Comment:
Does the invisible columns are needed ?

Reply:
Duplication, My bad. For others, added to display currency symbol!